src/bots/ama/subcommands/listEvents.js

```diff
@@ -0,0 +1,39 @@
+import client from '../../../client';
+import EventModel from '../models/Event';
+import { partition }  from '../../../utils/index';
+
+const listEvents = {
+  usage: "List all events",
+  example: "list",
+
+  handle: (args) => {
+    EventModel.find({}).sort({date: 'asc'}).then(a => client.message.channel.send(formatEvents(a)));
+  }
+}
+
+let formatEvents = (events) => {
+  let currentDate = new Date();
+  let splitDate = partition(events, e => e.date > currentDate);
+
+  let upcomingEvent = "-={Upcoming Event}=-\n" + splitDate[0][0].date.toDateString() + ": " + splitDate[0][0].title + "\n";
```

Comment:
can we just underline and bold these titles normally instead of the weird punctuation? I think it's something like

```
__**Upcoming Events**__
```